### PR TITLE
added send_final_comment

### PR DESCRIPTION
**Pull request by mec07**

### CODE REVIEW

**Comment by mec07** [comment]
This is working. I guess I could add a test. What do you think, @nicwest ?

File: pyticketswitch/send_method.py [hunk 1]
```diff
@@ -14,18 +14,20 @@ class SendMethod(JSONMixin, object):
             self print a voucher
         self_print_voucher_url (str): url for the client to download their
             voucher from
+        send_final_comment (str): comment about dispatching tickets
```
Comment (nicwest) [inline pyticketswitch/send_method.py]: I would rename this to `final_comment` it's already in the `SendMethod` class so the prefix is unnecessary

Comment (mec07) [inline pyticketswitch/send_method.py]: The white label was actually looking for something called "despatch_final_comment" (from the XML) maybe we should call it that as it is just specifically for tickets sent by post. What do you think?

Comment (nicwest) [inline pyticketswitch/send_method.py]: don't worry about the WL. we can change that if needed.  `final_comment` is the correct term here.

**Comment by nicwest** [comment]
would be good to add a test for this

https://github.com/ingresso-group/pyticketswitch/blob/master/tests/test_send_method.py

you can probably just append it to the existing test for from_api_data

**Pull request merged**